src/pages/overlay/components/overlay/Overlay.tsx

```diff
@@ -135,13 +135,11 @@ export default function Overlay() {
     return () => removeSleepListener("wake", callback);
   }, [addSleepListener, removeSleepListener]);
 
-  // Handle body clicks, dismissing the overlay if the user clicks outside of it
-  const bodyClick = useCallback((e: MouseEvent) => {
+  const visibleUnderCursor = (e: MouseEvent) => {
```

Comment:
Could we move this into a util? Perhaps `utils/dom.ts`? Where it is currently, it will change identity every time React re-renders, and following the rules of hooks it should then be a dependency of your `useCallback`s, which'd then mean they're recomptued every re-render as well.

Reply:
Thanks for the feedback 🙂 I've updated the method to return the element and moved it into a separate module.